src/components/standalone/openvpn_rw/CreateOrEditRWServerDrawer.vue

```diff
@@ -0,0 +1,646 @@
+<!--
+  Copyright (C) 2023 Nethesis S.r.l.
+  SPDX-License-Identifier: GPL-3.0-or-later
+-->
+
+<script setup lang="ts">
+import { ref, watch } from 'vue'
+import {
+  MessageBag,
+  validateIp4Cidr,
+  validateIpAddress,
+  validateIpAddressOrFQDN,
+  validatePort,
+  validateRequired,
+  type validationOutput
+} from '@/lib/validation'
+import {
+  NeSideDrawer,
+  NeInlineNotification,
+  NeToggle,
+  NeTextInput,
+  NeButton,
+  getAxiosErrorMessage,
+  NeTooltip,
+  NeCombobox,
+  NeFormItemLabel,
+  NeSkeleton,
+  type NeComboboxOption,
+  NeRadioSelection
+} from '@nethserver/vue-tailwind-lib'
+import { useI18n } from 'vue-i18n'
+import type { RWServer } from '@/views/standalone/vpn/OpenvpnRoadWarriorView.vue'
+import NeMultiTextInput, { type KeyValueItem } from '../NeMultiTextInput.vue'
+import { ValidationError, ubusCall } from '@/lib/standalone/ubus'
+
+const props = defineProps<{
+  isShown: boolean
+  itemToEdit?: RWServer
+  instanceName?: string
+}>()
+
+const { t } = useI18n()
+
+const emit = defineEmits(['close', 'add-server', 'edit-server'])
+
+const loading = ref(false)
+const isSavingChanges = ref(false)
+const error = ref({
+  notificationTitle: '',
+  notificationDescription: '',
+  notificationDetails: ''
+})
+const validationErrorBag = ref(new MessageBag())
+const publicIpOrHostnameValidationErrors = ref<string[]>([])
+const pushCustomNetworkRoutesValidationErrors = ref<string[]>([])
+
+const showAdvancedSettings = ref(false)
+
+// form fields
+const enabled = ref(true)
+const name = ref('')
+const userDatabase = ref('main')
+const createAccountForAllUsers = ref(false)
+const authMode = ref('username_password')
+const mode = ref('routed')
+const vpnNetwork = ref('')
+const publicIpOrHostname = ref<string[]>([])
+const protocol = ref('udp')
+const port = ref('')
+const routeTrafficThroughVpn = ref(false)
+const pushCustomNetworkRoutes = ref<string[]>([])
+const clientToClientNetworkTraffic = ref(false)
+const compression = ref('disabled')
```

Comment:
Value for disabled is an empty string:
```suggestion
const compression = ref('')
```

Reply:
For this issue i've simply changed line 363 in the drawer component, i forgot to change the correct value to associate with the default state